app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  def facebook
+    # You need to implement the method below in your model (e.g. app/models/user.rb)
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)